website/static/js/fileViewTreebeard.js

```diff
@@ -63,11 +63,13 @@ function FileViewTreebeard(data) {
             this.options.showTotal = Math.floor(containerHeight / this.options.rowHeight) + 1;
             this.redraw();
         },
-        lazyLoadOnLoad: function(tree) {
+        lazyLoadOnLoad: function(tree, event) {
             var tb = this;
             Fangorn.DefaultOptions.lazyLoadOnLoad.call(tb, tree);
             Fangorn.Utils.findCurrentFileID.call(tb, tree, window.contextVars.node.id, window.contextVars.file);
-            Fangorn.Utils.scrollToFile.call(tb, tb.currentFileID);
+            if(!event) { 
```

Comment:
@caneruguz To clarify, `event` is only a non-null value if the user clicks on a folder?

It would be good to add a comment here, as this fix is not obvious.